### DISCUSSION TITLE
MY Computer setup for mining

### DISCUSSION THREAD

**Discussion by udayantha1989** [Chia-Network/chia-blockchain#3395] [General]
I am planning tioget 16TB external WD brand HD. Computer configuration going to purchase is Core i5 4th or 6th. 4 GB RAM. include Nvmi 1TB SSD too.

So is it possible to mine chia?

**Comment by pluto2021** [comment]
It is like buying expensive lottery tickets for the moment. Everyone is waiting for the pool to officially open.  Would suggest planning for the computer as a dual use machine so that if this thing does not pan out, you can use it for some other purposes, or hobbies, etc.

**Reply by udayantha1989** [reply]
thnx
